lib/Room.php

```diff
@@ -48,6 +49,8 @@ class Room {
 	private $token;
 	/** @var string */
 	private $name;
+	/** @var string */
+	private $password;
```

Comment:
Where is this value ever read from the database?

Reply:
In the manager